app/components/pages/SubmissionWizard/steps/Disclosure/schema.js

```diff
@@ -0,0 +1,11 @@
+import * as yup from 'yup'
+
+const schema = yup.object().shape({
+  submitterSignature: yup.string().required('Your name is required'),
+  disclosureConsent: yup
+    .bool()
+    .required()
+    .oneOf([true], 'We are unable to proceed without your consent'),
+})
+
+export { schema }
```

Comment:
It's worth writing a test for this, as this is what is used to decide whether the user is allowed to Submit or not.

Reply:
I know that I used to bring this up during standups a fair bit, but at the risk of becoming annoying about this, perhaps now is a good time for us to revisit the creation of a testing strategy? #255 :pray: 

Testing the schema would be a good way of catching validation bugs :+1: More than happy to do this.

But there are plenty of other ways our app could break too, which we're not covering with tests e.g. progressing through the submission wizard correctly. And the layout bug wasn't the first time that breaking changes have made their way into `develop`: https://github.com/elifesciences/elife-xpub/issues?q=is%3Aissue+label%3ABug+is%3Aclosed, which we should probably start being more strict with, since we're coming up to first release?

Then we could start making certain kinds of tests a requirement for each PR, so that we don't keep saying:
> #### How has this been tested?
>
> Visually / manually

Reply:
This is a discussion we should have with everyone else. I suggest creating an issue for this, including the discussion.

Reply:
Yeah I'll comment more thoroughly in #255 a bit later today

Reply:
I've pushed a `schema.test.js` file. Lemme know what you think & if there are other tests you think should be in this PR? :smile: